examples/capi-quick-start/docker-cluster-class.yaml

```diff
@@ -4,52 +4,30 @@
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: ClusterClass
 metadata:
-  name: capd-quick-start
+  labels:
+    cluster.x-k8s.io/provider: docker
+  name: docker-quick-start
 spec:
   controlPlane:
     machineInfrastructure:
       ref:
         apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
         kind: DockerMachineTemplate
-        name: capd-quick-start-control-plane
+        name: aws-quick-start-control-plane
     ref:
       apiVersion: controlplane.cluster.x-k8s.io/v1beta1
       kind: KubeadmControlPlaneTemplate
-      name: capd-quick-start-control-plane
+      name: docker-quick-start-control-plane
   infrastructure:
     ref:
       apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
       kind: DockerClusterTemplate
-      name: capd-quick-start-cluster
+      name: docker-quick-start-cluster
   patches:
     - external:
         discoverVariablesExtension: clusterconfigvars.capi-runtime-extensions
         generateExtension: clusterconfigpatch.capi-runtime-extensions
       name: cluster-config
-    - definitions:
-        - jsonPatches:
-            - op: add
-              path: /spec/template/spec/customImage
-              valueFrom:
-                template: ghcr.io/mesosphere/kind-node:{{ .builtin.machineDeployment.version }}
-          selector:
-            apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-            kind: DockerMachineTemplate
-            matchResources:
-              machineDeploymentClass:
-                names:
-                  - '*'
-        - jsonPatches:
-            - op: add
-              path: /spec/template/spec/customImage
-              valueFrom:
-                template: ghcr.io/mesosphere/kind-node:{{ .builtin.controlPlane.version }}
-          selector:
-            apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
-            kind: DockerMachineTemplate
-            matchResources:
-              controlPlane: true
-      name: customKinDNodeImageForWorkers
```

Comment:
Did you mean to remove this, not all images would work here?

Reply:
Fixed this too now.